package.json

```diff
@@ -37,7 +37,7 @@
     "babel-eslint": "^9.0.0",
     "babel-jest": "^23.6.0",
     "babel-loader": "^8.0.0",
-    "babel-plugin-emotion": "^10.0.9",
+    "@emotion/babel-plugin": "^11.0.0",
```

Comment:
Would be nice to move this into depsDependencies. I mean it's not the only one that should move but that would be a start.

(not to do in root package.json, there it's fine)

Reply:
Forget about this one, mistakenly took the package.json for a package one ;)